src/services/platform.service.js

```diff
@@ -39,7 +39,7 @@ export const PACKAGE_MANAGER_CMD = path.join(
   remote.app.getAppPath(),
   'node_modules/yarn/bin',
   formatCommandForPlatform(PACKAGE_MANAGER)
-);
+).replace('app.asar', 'app.asar.unpacked');
```

Comment:
Why is this required? Can you please give some details? 
What was the error message if you're not replacing it?

Reply:
Once unpacking yarn, it is placed under the directory \app.asar.unpacked\node_modules\.. instead of within the \app.asar\node_modules\.. asar file. Even though it was unpacked, code is still referencing it within the app.asar, so replacing the path.

Same issue [is discussed here](https://github.com/electron/electron/issues/6262). I elected to workaround since this was the only case I knew of, but we could instead use the `hazardous` package referenced in the discussion if desired.